fission/src/vite-env.d.ts

```diff
@@ -1,10 +1 @@
 /// <reference types="vite/client" />
-
-interface ImportMetaEnv {
-    readonly VITE_SYNTHESIS_SERVER_PATH: string
-    // more env variables...
-}
-
-interface ImportMeta {
-    readonly env: ImportMetaEnv
-}
```

Comment:
Do we still need to save this file?

Reply:
It was kept in prior to me using and we might use it in the future for other vite environment stuff, so I decided to keep it in for now.